backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/domain/mapper/SubscriptionMapper.java

```diff
@@ -13,9 +16,21 @@
 @Mapper(uses = { AlertTypeMapper.class })
 public interface SubscriptionMapper {
 
+	@Nullable
+	public default List<Subscription> fromEntity(@Nullable Iterable<SubscriptionEntity> subscriptions) {
+		if (subscriptions == null) { return null; }
+		return StreamSupport.stream(subscriptions.spliterator(), false).map(this::fromEntity).toList();
+	}
+	
 	@Nullable
 	Subscription fromEntity(@Nullable SubscriptionEntity subscription);
 
+	@Nullable
+	public default List<SubscriptionEntity> toEntity(@Nullable Iterable<Subscription> subscriptions) {
+		if (subscriptions == null) { return null; }
+		return StreamSupport.stream(subscriptions.spliterator(), false).map(this::toEntity).toList();
+	}	
```

Comment:
These two new `fromEntity` and `toEntity` methods that accept `Iterable` aren't necessary in my opinion because we can use the existing `fromEntity(SubscriptionEntity subscription)` with a stream pipeline. For example:

```java
allSubscriptionEntities.stream().map(subscriptionMapper::fromEntity).toList()
```